### PR TITLE
Create dosbox.xml

### PR DESCRIPTION
**Pull request by redranamber**

### CODE REVIEW

File: src/chrome/content/rules/dosbox.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<ruleset name="DOSbox">
+	<target host="dosbox.com"/>
+	<target host="www.dosbox.com"/>
+  <target host="vogons.org"/>
```
Comment (J0WI) [inline src/chrome/content/rules/dosbox.xml]: Please use tabs to indent lines.

**Comment by redranamber** [comment]
What do you mean about adding a comment?

**Comment by J0WI** [comment]
You should have a look at our (docs)[https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md] and the recommend tolls there.

**Comment by J0WI** [comment]
ping @redranamber

**Comment by redranamber** [comment]
Does that work for you?

**Comment by J0WI** [comment]
Yes, but please not them on top next time: https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#non-working-hosts

**Pull request merged**